### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by jjuli93**
## Description
Added navbar component
Added local storage service
Token handling on sign in and logout

## Screenshot
image

## Trello card
https://trello.com/c/cERMWbME/24-session-management-logout

### CODE REVIEW

File: wolox-books-vue/src/router.js [hunk 1]
```diff
@@ -2,14 +2,16 @@ import Vue from 'vue'
...
 
 Vue.use(Router)
 
 export default new Router({
   routes: [
     {
       path: '',
-      redirect: { name: 'signUp' }
+      name: 'home',
+      component: NavBar
```
Comment (FrankIglesias) [inline wolox-books-vue/src/router.js]: use require

File: wolox-books-vue/src/scss/application.scss [hunk 2]
```diff
@@ -4,3 +4,4 @@
 @import './buttons.scss';
 @import './margins.scss';
 @import './forms.scss';
+@import './text.scss';
```
Comment (FrankIglesias) [inline wolox-books-vue/src/scss/application.scss]: remove file extension here

File: wolox-books-vue/src/views/NavBar.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+  nav.navbar.row.middle.space-around.full-width
+    div.main.row.space-between
```
Comment (FrankIglesias) [inline wolox-books-vue/src/views/NavBar.vue:3]: remove divs

File: wolox-books-vue/src/views/NavBar.vue [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      img.logo(src='../assets/wolox-logo.png')
+      button.bold(@click="logOut") Logout
+</template>
+
+<script>
+import { LocalStorageService } from '../services/localStorageService'
+
+const localStorageService = new LocalStorageService()
```
Comment (FrankIglesias) [inline wolox-books-vue/src/views/NavBar.vue:11]: No need of a class here

File: wolox-books-vue/src/views/NavBar.vue [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    logOut () {
+      localStorageService.removeValue(localStorageService.SESSION_TOKEN)
+      this.$router.push({ name: 'signIn' })
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../scss/colors.scss';
```
Comment (FrankIglesias) [inline wolox-books-vue/src/views/NavBar.vue]: remove extension

File: wolox-books-vue/src/views/NavBar.vue [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+  .navbar {
+    height: 60px;
+    border-top: 4px solid $celurean;
+    box-shadow: 0px 10px 5px -6px $wild-sand;
+  }
+
+  .main {
+    width: 1132px;
```
Comment (FrankIglesias) [inline wolox-books-vue/src/views/NavBar.vue:33]: width 100vw or 100%

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: wolox-books-vue/src/views/NavBar.vue (hunk 3 above)
Comment (jjuli93) [reply wolox-books-vue/src/views/NavBar.vue:3]: the div is to have the logo and the logout button inside the inner container

File: wolox-books-vue/src/views/NavBar.vue (hunk 6 above)
Comment (jjuli93) [reply wolox-books-vue/src/views/NavBar.vue:33]: the design has an inner container of that width

File: wolox-books-vue/src/views/NavBar.vue (hunk 4 above)
Comment (jjuli93) [reply wolox-books-vue/src/views/NavBar.vue:11]: i couldnt find the localStorage file so i used the typescript version which is a class

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**